metadata.rb

```diff
@@ -6,7 +6,7 @@
 source_url        'https://github.com/sous-chefs/consul'
 issues_url        'https://github.com/sous-chefs/consul/issues'
 chef_version      '>= 13.4'
-version           '4.0.3'
+version           '4.0.4'
```

Comment:
Please don't bump the version in the PR. We'll do that separately.

Reply:
Thanks, done.